example/queue/submit_taiwania3_gnu.job

```diff
@@ -19,13 +19,14 @@
 ##SBATCH --mail-user=EMAIL_ADDRESS                          # Where to send mail.  Set this to your email address
 #SBATCH --exclude=cpn[3001-3120,3241-3360]                  # Exclude large-memory nodes
 
-LOG_FILE=log_taiwania_III_gnu_9.4.0
+LOG_FILE=log_taiwania_III_gnu_13.2.0
 
 module purge
-module load compiler/gcc/9.4.0 OpenMPI/4.1.1
+module use /home/d07222009/module_CALAB
+module load gcc/13.2.0 gnu_13.2.0/fftw/3.3.10 gnu_13.2.0/gsl/2.8.0 gnu_13.2.0/hdf5/1.14.4 gnu_13.2.0/openmpi/5.0.5  gnu_13.2.0/openucx/1.18.0
 module list >> $LOG_FILE
```

Comment:
Now, it recorded the module information in `slurm-*.out`.
How about `module list 1>>$LOG_FILE 2>&1` to save in the same `log` and be more consistent with other job scripts?

Reply:
Modified~